src/components/prompts/Share.vue

```diff
@@ -1,156 +1,364 @@
 <template>
```

Comment:
We need to refactor this file and shorten it drastically.

Reply:
Agreed

Reply:
Did you open an issue for it?

Reply:
No. Working on it in feature/beautify-share

Reply:
#140